cvat-data/src/ts/cvat-data.ts

```diff
@@ -175,12 +201,14 @@ export class FrameDecoder {
                 // it was other chunk
                 this.requestedChunkToDecode.onReject(new RequestOutdatedError());
             }
-        } else if (this.chunkIsBeingDecoded === null || this.chunkIsBeingDecoded.start !== start) {
+        } else if (this.chunkIsBeingDecoded === null ||
+            !this.arraysEqual(frameNumbers, this.requestedChunkToDecode.frameNumbers)
```

Comment:
From the previous condition `this.requestedChunkToDecode` is null, but here you use `this.requestedChunkToDecode.frameNumbers`

Reply:
Fixed